### DISCUSSION TITLE
image-tagging bundle not working with Pimcore 10.1.1

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#10384] [Q&A / Support]
Hi there,

it is not possible to install image-tagging bundle with Pimcore v10.1.1.
i received following error messages:

_$ composer require pimcore/image-tagging --with-all-dependencies
Using version ^0.1.3 for pimcore/image-tagging
./composer.json has been updated
Running composer update pimcore/image-tagging --with-all-dependencies
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires pimcore/image-tagging ^0.1.3 -> satisfiable by pimcore/image-tagging[v0.1.3].
    - pimcore/image-tagging v0.1.3 requires pimcore/pimcore ^5.4 || ^6.0 -> found pimcore/pimcore[v5.4.0, ..., v5.8.9, v6.0.0, ..., v6.9.6] but it conflicts with your root composer.json require (^10.1.1).

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

Thank you

Hakan

**Comment by fashxp** [comment]
this bundle is not compatible to pimcore X yet ...

**Reply by exploit-Unicor** [reply]
Hello,

Is there a plan to make it compatible with the latest version of Pimcore (10.5 for now)?
Thanks in advance for your answer.

Fabrice

**Comment by arunvasisht** [comment]
Is there a plan to update this? or is there an alternative to this extension?

Please advise.
